app/api/browser/agent/route.ts

```
@@ -0,0 +1,55 @@
+      await page.goto(startUrl, { waitUntil: "domcontentloaded", timeout: 20000 });
+      return await page.act(task);
+    });
+
+    // Check if result is an object with success property, otherwise assume success
+    const isObject = typeof result === "object" && result !== null && !Array.isArray(result);
+    const hasSuccess = isObject && "success" in result;
+    
+    const out = typeof result === "string" ? result : JSON.stringify(result);
+    
+    return NextResponse.json({
+      success: hasSuccess ? Boolean((result as any).success) : true,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)